src/engraving/playback/playbackeventsrenderer.cpp

```diff
@@ -95,16 +95,6 @@ void PlaybackEventsRenderer::renderRestEvents(const Ms::Rest* rest, mpe::Playbac
 }
 
 void PlaybackEventsRenderer::renderArticulations(const Ms::Chord* chord, PlaybackContext&& ctx, mpe::PlaybackEventList& result) const
-{
```

Comment:
no idea what's happening here, I didn't touch these files, and have tried to update locally with the version from upstream/master